graphql/resolvers.go

```diff
@@ -1822,13 +1822,15 @@ func (r *queryResolver) TaskTestSample(ctx context.Context, tasks []string, test
 	if len(tasks) == 0 {
 		return nil, nil
 	}
-	t, err := task.Find(task.ByIds(tasks))
+	t, err := task.FindAll(task.ByIds(tasks))
```

Comment:
"t" implies one task, not many. Can we rename this to tasks? (Also helps us avoid the `for t := range t` thing below)